eta/core/annotations.py

```diff
@@ -1,6 +1,10 @@
 '''
 Core utilities for rendering annotations on media.
 
+@todo support for rendering video-level attributes
```

Comment:
Questioning whether we plan to keep functionality like this in ETA.

Reply:
The goal is for ETA users to be able to render a  `VideoLabels` instance on a video, which should help popularize ETA formats.